great_expectations/data_context/data_context.py

```diff
@@ -852,6 +852,9 @@ def _compile_evaluation_parameter_dependencies(self):
         self._evaluation_parameter_dependencies = {}
         for key in self.stores[self.expectations_store_name].list_keys():
             expectation_suite = self.stores[self.expectations_store_name].get(key)
+            if not expectation_suite:
+                continue
```

Comment:
I liked the earlier version that had a warning here. But I think we can include that in updates to the store API later.